applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp

```diff
@@ -0,0 +1,48 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez
+
+#include "custom_utilities/potential_flow_utilities.h"
+
+namespace Kratos {
+namespace PotentialFlowUtilities {
+template <int Dim, int NumNodes>
+array_1d<double, NumNodes> GetPotentialOnNormalElement(const Element& rElement)
+{
+    const int kutta = rElement.GetValue(KUTTA);
+    array_1d<double, NumNodes> potentials;
+
+    const auto r_geometry = rElement.GetGeometry();
+
+    if (kutta == 0) {
+        for (unsigned int i = 0; i < NumNodes; i++) {
+            potentials[i] = Geometry[i].FastGetSolutionStepValue(VELOCITY_POTENTIAL);
```

Comment:
Is this working in your machine? Seems like it should be `r_geometry[i].Fast...`

```suggestion
            potentials[i] = r_geometry[i].FastGetSolutionStepValue(VELOCITY_POTENTIAL);
```

Reply:
You are right, I just committed Philipp's suggestion through Github without checking